integration/model_splunk_platform_integration.go

```diff
@@ -0,0 +1,30 @@
+package integration
+
+type SplunkPlatformIntegration struct {
```

Comment:
From what I can tell, this implements the public API described here: https://dev.splunk.com/observability/reference/api/integrations/latest